pkg/agent/openflow/client_test.go

```diff
@@ -948,8 +952,8 @@ func Test_client_InstallEndpointFlows(t *testing.T) {
 				proxy.NewBaseEndpointInfo(ep2IPv6, "", "", 80, true, true, false, false, nil),
 			},
 			expectedFlows: []string{
-				"cookie=0x1030000000000, table=EndpointDNAT, priority=200,tcp6,reg4=0x20050/0x7ffff actions=ct(commit,table=AntreaPolicyEgressRule,zone=65510,exec(nat(dst=[fec0:10:10::100]:80),set_field:0x10/0x10->ct_mark,move:NXM_NX_REG0[0..3]->NXM_NX_CT_MARK[0..3]))",
```

Comment:
I am confused why the test succeed before this patch?

Reply:
I changed the code of generating flow string. This is not the same with the actual flow string dumped from OVS, so I changed the code.